### PR TITLE
Allow modifying bestuursorganen (niet decretaal), view only on decretaal

### PR DESCRIPTION
**Pull request by ruben-bottu**
image
image
For app-lokaal-mandatenbeheer, the branch `karel/lmb-115-create-bestuursorgaan` should be checked out for the same result

### CODE REVIEW

File: tests/unit/controllers/mandatenbeheer/bestuursorganen/edit-test.js
Comment (bfidlers) [inline tests/unit/controllers/mandatenbeheer/bestuursorganen/edit-test.js:1]: I'm not really a fan of including empty tests ...

File: app/templates/mandatenbeheer/fracties/edit.hbs
Comment (bfidlers) [inline app/templates/mandatenbeheer/fracties/edit.hbs:1]: Not sure why this file is edited, doesn't really seem to have anything to do with this PR?

File: app/templates/mandatenbeheer/bestuursorganen/edit.hbs [hunk 1]
```diff
@@ -1,3 +1,18 @@
 {{page-title "Edit Bestuursorgaan"}}
+<AuToolbar @border="bottom" @size="large" as |Group|>
+  <Group>
+    <AuHeading @skin="2">Bewerk fractie</AuHeading>
```
Comment (bfidlers) [inline app/templates/mandatenbeheer/bestuursorganen/edit.hbs:4]: Headers are wrong here.

**Review by bfidlers — APPROVED** [review]
Looks good and can certainly be merged into the Karel/lmb-116 branch, but probably best to rebase this first in the current structure of the bestuursorganen subpage before merging to master.

**Review by bfidlers — CHANGES_REQUESTED** [review]
Needs to be re-reviewed

**Review by Rahien — CHANGES_REQUESTED** [review]
could we get a screenshot of the end result before approving?

File: app/models/bestuursorgaan.js [hunk 2]
```diff
@@ -43,6 +51,19 @@ export default class BestuursorgaanModel extends Model {
   })
   bevat;
 
+  async classificatieUri() {
+    const bestuursorgaan = await this.isTijdsspecialisatieVan;
+    return bestuursorgaan
+      ? await bestuursorgaan.get('classificatie.uri')
+      : await this.classificatie.get('uri');
```
[the remark this answers is not in the document]
Comment (ruben-bottu) [reply app/models/bestuursorgaan.js:58]: What about bestuursorgaan? I guess it doens't have to be awaited anymore

[the remark this answers is not in the document]
Comment (Rahien) [reply app/models/bestuursorgaan.js:58]: it already is on line 55

**Review by Rahien — APPROVED** [review]

**Pull request merged**